src/parser.rs

```diff
@@ -382,11 +382,10 @@ impl CommentParser<Comments> {
                     }
 
                     for (span, line, idx) in delayed_fallthrough.drain(..) {
-                        if let Some(rev) = self
-                            .comments
-                            .revisioned
-                            .values_mut()
-                            .find(|rev| rev.error_matches[idx].line == line)
```

Comment:
This was the cause. Only the revision each comment was added to was guaranteed to have the index.

Reply:
oh right